src/components/pages/books/books.css

```diff
@@ -137,8 +184,8 @@
   border-top: 2px solid grey;
   margin-top: 10px;
   margin-bottom: 20px;
-  width: 94%;
-  margin-left: 3%;
+  width: 85%;
+  margin-left: 5%;
```

Comment:
- [ ] Also same here please provide` border: 1px solid #e8e8e8` to the horizontal line as you gave 2px solid grey.

Reply:
Thank you for the feeadback